### PR TITLE
Add libraries in Choolai

### PR DESCRIPTION
**Pull request by srravya**

### CODE REVIEW

File: geo-data/libraries_new.geojson [hunk 1]
```diff
@@ -1333,6 +1333,121 @@
...
+                    80.259066,
+                    13.089999
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "addr:city": "Chennai",
+                "addr:country": "IN",
+                "addr:housenumber": "1",
+                "addr:street": "Rottler Streete, Choolai",
```
Comment (eternaltyro) [inline geo-data/libraries_new.geojson]: Rottler Street --> Rottler Street ?

File: geo-data/libraries_new.geojson [hunk 2]
```diff
@@ -1333,6 +1333,121 @@
...
+                    80.260158,
+                    13.030056
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "addr:city": "Chennai",
+                "addr:country": "IN",
+                "addr:housenumber": "",
+                "addr:street": "Warren Road, Mylapore",
```
Comment (eternaltyro) [inline geo-data/libraries_new.geojson:1441]: Warren road or St. Mary's Road?

Comment (srravya) [inline geo-data/libraries_new.geojson:1441]: So the sign board and our list says St. Mary's road but It is actually on a left cut (Warren road) from St. Mary's road. So which one do we pick?

File: geo-data/libraries_new.geojson (hunk 1 above)
Comment (srravya) [inline geo-data/libraries_new.geojson]: My bad. corrected

File: geo-data/libraries_new.geojson (hunk 2 above)
Comment (eternaltyro) [inline geo-data/libraries_new.geojson:1441]: Whichever road it is on. Our on-site survey data is final. I'm merging this.

**Pull request merged**